<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mayo::MainWindow</class>
 <widget class="QMainWindow" name="Mayo::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mayo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stack_Main">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_MainHome">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_MainHome" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_AppIcon">
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../mayo.qrc">:/images/appicon_128.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignHCenter|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="WidgetHomeFiles" name="widget_HomeFiles" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_MainControl">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_Main">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="widget_Left" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_LeftContents" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget_LeftHeader" native="true">
                 <layout class="QHBoxLayout" name="Layout_WidgetLeftHeader">
                  <property name="spacing">
                   <number>1</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>4</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QComboBox" name="combo_LeftContents">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>1</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>22</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>Model tree</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Opened documents</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>File system</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="Line" name="line_SeparatorComboLeftContents">
                    <property name="frameShadow">
                     <enum>QFrame::Plain</enum>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="btn_CloseLeftSideBar">
                    <property name="toolTip">
                     <string>Close Left Side Bar</string>
                    </property>
                    <property name="icon">
                     <iconset>
                      <normaloff>:/images/themes/classic/left-arrow-cross_16.png</normaloff>:/images/themes/classic/left-arrow-cross_16.png</iconset>
                    </property>
                    <property name="autoRaise">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QStackedWidget" name="stack_LeftContents">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="page_ModelTree">
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QSplitter" name="splitter_ModelTree">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <widget class="WidgetModelTree" name="widget_ModelTree" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>1</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                     <widget class="WidgetPropertiesEditor" name="widget_Properties" native="true"/>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_OpenedDocuments">
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QListView" name="listView_OpenedDocuments">
                     <property name="horizontalScrollBarPolicy">
                      <enum>Qt::ScrollBarAlwaysOff</enum>
                     </property>
                     <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                     </property>
                     <property name="textElideMode">
                      <enum>Qt::ElideMiddle</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="page_FileSystem">
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="WidgetFileSystem" name="widget_FileSystem" native="true"/>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_Right" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_ControlGuiDocuments" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolButton" name="btn_PreviousGuiDocument">
                 <property name="icon">
                  <iconset>
                   <normaloff>:/images/themes/classic/back_32.png</normaloff>:/images/themes/classic/back_32.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_NextGuiDocument">
                 <property name="icon">
                  <iconset>
                   <normaloff>:/images/themes/classic/next_32.png</normaloff>:/images/themes/classic/next_32.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="combo_GuiDocuments">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="sizeAdjustPolicy">
                  <enum>QComboBox::AdjustToContents</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line">
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_CloseGuiDocument">
                 <property name="icon">
                  <iconset>
                   <normaloff>:/images/themes/classic/cross_32.png</normaloff>:/images/themes/classic/cross_32.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="Line" name="line_2">
                 <property name="frameShadow">
                  <enum>QFrame::Plain</enum>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_Spacer" native="true">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>1</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_MouseCoords" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_PosX">
                      <property name="text">
                       <string>X=</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_ValuePosX">
                      <property name="text">
                       <string>?</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_PosY">
                      <property name="text">
                       <string>Y=</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_ValuePosY">
                      <property name="text">
                       <string>?</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_PosZ">
                      <property name="text">
                       <string>Z=</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_ValuePosZ">
                      <property name="text">
                       <string>?</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stack_GuiDocuments">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Display">
    <property name="title">
     <string>&amp;Display</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Display"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WidgetModelTree</class>
   <extends>QWidget</extends>
   <header>widget_model_tree.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetFileSystem</class>
   <extends>QWidget</extends>
   <header>widget_file_system.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPropertiesEditor</class>
   <extends>QWidget</extends>
   <header>widget_properties_editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetHomeFiles</class>
   <extends>QWidget</extends>
   <header>widget_home_files.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../mayo.qrc"/>
 </resources>
 <connections/>
</ui>
